src/Actions/CreateInvoice.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Supplycart\Xero\Actions;
+
+use Supplycart\Xero\Contracts\ShouldCheckConnection;
+
+class CreateInvoice extends Action implements ShouldCheckConnection
+{
+    public function handle(array $data)
+    {   
+        $this->log(__CLASS__ . ': START');
+
+        $response = $this->xero->client->post('https://api.xero.com/api.xro/2.0/Invoices', [
```

Comment:
why not using our xero package function? if there is none can add the function in the package

Reply:
ignore this, i wrong see it